openmoc/compatible/opencg_compatible.py

```diff
@@ -374,21 +374,29 @@ def get_opencg_cell(openmoc_cell):
   name = openmoc_cell.getName()
   opencg_cell = opencg.Cell(cell_id, name)
 
-  fill = openmoc_cell.getFill()
-  if (openmoc_cell.getType == openmoc.MATERIAL):
+  if (openmoc_cell.getType() == openmoc.MATERIAL):
+    fill = openmoc_cell.getFillMaterial()
     opencg_cell.fill = get_opencg_material(fill)
   elif (openmoc_cell.getType() == openmoc.FILL):
+    fill = openmoc_cell.getFillUniverse()
     if isinstance(fill, openmoc.Lattice):
       opencg_cell.fill = get_opencg_lattice(fill)
     else:
       opencg_cell.fill = get_opencg_universe(fill)
 
+  if openmoc_cell.isRotated():
+    rotation = openmoc_cell.getRotation(3)
+    opencg_cell.rotation = rotation * -1
```

Comment:
Haha, it was a little hack while I was trying to figure things out. I've removed it now as I've straightened out things in the roll-pitch-yaw convention of the rotation matrix below.